grails-app/services/org/pih/warehouse/product/ProductService.groovy

```diff
@@ -282,10 +282,11 @@ class ProductService {
      * @param category
      * @param catalogs
      * @param tags
+     * @param glAccounts
      * @param params
      * @return
      */
-    List<Product> getProducts(List<Category> categories, List<ProductCatalog> catalogsInput, List<Tag> tagsInput, boolean includeInactive, Map params) {
+    List<Product> getProducts(List<Category> categories, List<ProductCatalog> catalogsInput, List<Tag> tagsInput, List<GlAccount> glAccounts, boolean includeInactive, Map params) {
```

Comment:
I am starting to dislike these parameters more with each new one. I think we could think about refactoring this one.